Arc/Arc.js

```diff
@@ -92,15 +92,37 @@ const ArcBase = kind({
 		 * @default 9
 		 * @public
 		 */
-		strokeWidth: PropTypes.number
+		strokeWidth: PropTypes.number,
+
+		/**
+		 * Specifies if the svg  can become the target of pointer events.
+		 *
+		 * @type {number}
+		 * @default 'none'
+		 * @public
+		 */
+		svgPointerEvents: PropTypes.oneOf(['none', 'auto']),
+
+		/**
+		 * A reference to the current svg.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		svgRef: PropTypes.object
```

Comment:
For the `ref`, we usually name it as `componentRef`. Could you rename the `svgRef` as the `componentRef`?